src/SetupRepository.js

```diff
@@ -22,14 +22,14 @@ import { SetupToken } from './SetupToken';
 import { SetupWebDAV } from './SetupWebDAV';
 
 const supportedProviders = [
-    { provider: "filesystem", description: "Filesystem", component: SetupFilesystem },
+    { provider: "filesystem", description: "Local directory or network share", component: SetupFilesystem },
     { provider: "gcs", description: "Google Cloud Storage", component: SetupGCS },
-    { provider: "s3", description: "Amazon S3, IDrive e2, Minio, Wasabi, etc.", component: SetupS3 },
+    { provider: "s3", description: "Amazon S3 and S3-compatible Storage Providers", component: SetupS3 },
```

Comment:
"Amazon S3 or Compatible" perhaps?

Reply:
If we want the text to be as short as possible, what you suggest is great. However, if we have some space, what if we just say "Storage Providers Compatible with S3"? That's a catchall for all S3-compatible providers, including AWS itself. This may work if combined with the proposed changes at https://github.com/kopia/htmlui/pull/71

LMK and Ill make the relevant edits.

Reply:
"S3 Compatible" is an option too and is quite short =)